src/server/scripts/Events/Anniversary/boss_lord_kazzak_anniversary.cpp

```diff
@@ -0,0 +1,235 @@
+/*
+ * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ScriptMgr.h"
+#include "ScriptedCreature.h"
+#include "SpellAuraEffects.h"
+#include "SpellAuras.h"
+#include "SpellScript.h"
+
+enum LordKazzakSpells
+{
+    SPELL_MARK_OF_KAZZAK_SELECTOR  = 243737,
+    SPELL_MARK_OF_KAZZAK_PERIODIC  = 243723,
+    SPELL_MARK_OF_KAZZAK_DAMAGE    = 243725,
+    SPELL_MARK_OF_KAZZAK_EXPLOSION = 243726,
+    SPELL_VOID_BOLT                = 243713,
+    SPELL_THUNDERCLAP              = 243715,
+    SPELL_SHADOW_BOLT_VOLLEY       = 243712,
+    SPELL_FRENZY                   = 156598 
+};
+
+enum LordKazzakTexts
+{
+    SAY_AGGRO       = 0,
+    SAY_KILL        = 1,
+    SAY_FRENZY      = 2
+};
+
+enum LordKazzakEvents
+{
+    EVENT_MARK_OF_KAZZAK = 1,
+    EVENT_VOID_BOLT,
+    EVENT_THUNDERCLAP,
+    EVENT_SHADOW_BOLT_VOLLEY
+};
+
+// 121818 - Lord Kazzak
+struct boss_lord_kazzak_anniversary : public WorldBossAI
+{
+    using WorldBossAI::WorldBossAI;
+
+    void JustDied(Unit* killer) override
+    {
+        WorldBossAI::JustDied(killer);
+    }
+
+    void KilledUnit(Unit* victim) override
+    {
+        if (victim->GetTypeId() == TYPEID_PLAYER)
+            Talk(SAY_KILL, victim);
```

Comment:
I recommend adding a timestamp check to prevent Talk spamming.

Reply:
we can do that in creature_text table at some point